sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/EditContactPage.java

```diff
@@ -27,6 +27,7 @@
   public static final By REPORT_DATE = By.cssSelector("#reportDateTime input");
   public static final By DISEASE_COMBOBOX =
       By.cssSelector(".v-verticallayout [location='disease'] [role='combobox'] div");
+  public static final By DISEASE_VALUE = By.xpath("//div[text()='COVID-19 ']");
```

Comment:
we cannot create locators based on specific data, because this is not reusable. Also, it wouldn't be stable if you have this value in other places.